Makefile

```diff
@@ -70,26 +70,12 @@ upload: archive                                              ## Upload binary
 	aws s3 sync build/s3/ s3://$(BUCKET_NAME)/
 .PHONY: upload
 
-vet-check: build_deps                                        ## Verify vet compliance
-ifeq ($(shell go vet -all . | wc -l | tr -d '[:space:]'), 0)
-	@printf "ok\tall files passed go vet\n"
-else
-	@printf "error\tsome files did not pass go vet\n"
-	@go vet -all . 2>&1
-	@exit 1
-endif
-
+vet-check:                                                   ## Verify vet compliance
+	@go vet -all ./...
```

Comment:
Default output of `go vet` seems fine to me. No need to print extra error messages.

![image](https://user-images.githubusercontent.com/610129/74116662-97a18c80-4b82-11ea-84a7-ec587ea5114c.png)